lib/ecto_job/migrations.ex

```diff
@@ -180,7 +182,7 @@ defmodule EctoJob.Migrations do
     end
 
     @doc """
-    Rollback updates from job queue table with the given ecto job version and name.
+    Rollbacks updates from job queue table with the given ecto job version and name.
```

Comment:
I don't think this is correct grammar.

Reply:
> I don't think this is correct grammar.

Yup. You're right.

Reply:
This should be 'Rolls back'. :)

Reply:
@DavidOliver Please check again. Thanks.

Reply:
@kianmeng Sorry - how do you mean? I don't see any further commits to check? If you mean to check I'm correct, see https://www.merriam-webster.com/dictionary/rollback

The `up` function above `down` should also be 'Upgrades' instead of 'Upgrade' to be consistent with the other functions' descriptions.

Reply:
@kianmeng Oh, I think you've clobbered the commit. I see you currently have it as 'Roll back', but it should be 'Rolls back' to be consistent with the other function descriptions.

Reply:
@DavidOliver Noted. I will not squashed commits. If I understand correctly:

- `Roll back` -> `Rolls back`
- `Upgrades` -> `Upgrade`

Reply:
`Upgrade` -> `Upgrades` :)

Reply:
Done.

![image](https://user-images.githubusercontent.com/134518/147252550-7d1c3509-208e-46cd-b9e0-a71209fd8bfb.png)